docs/architecture.md

```diff
@@ -133,7 +133,7 @@ The supported KV stores for the hash ring are:
 
 * [Consul](https://www.consul.io)
 * [Etcd](https://etcd.io)
-* Gossip [memberlist](https://github.com/hashicorp/memberlist) (experimental)
+* Gossip [memberlist](https://github.com/hashicorp/memberlist)
```

Comment:
If this is no longer experimental this might be worth an entry in the changelog I guess?